From 06cdcd2a9dd6b46b7fbb5b3ae0a62bdb7ee6c402 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 27 Jun 2011 17:48:18 -0700
Subject: [PATCH] Config.in.arch: Free UCLIBC_HAS_FPU setting from depending on UCLIBC_HAS_FLOATS

UCLIBC_HAS_FLOATS and UCLIBC_HAS_FPU are denoting two different aspects.
UCLIBC_HAS_FLOATS covers the floating point operations which has nothing
to do if you have FPU or not.

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 extra/Configs/Config.in.arch |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/extra/Configs/Config.in.arch b/extra/Configs/Config.in.arch
index 5f7a2b0..c1ca390 100644
--- a/extra/Configs/Config.in.arch
+++ b/extra/Configs/Config.in.arch
@@ -133,7 +133,6 @@ config UCLIBC_HAS_FLOATS
 
 config UCLIBC_HAS_FPU
 	bool "Target CPU has a floating point unit (FPU)"
-	depends on UCLIBC_HAS_FLOATS
 	default y
 	help
 	  If your target CPU does not have a Floating Point Unit (FPU) or a
-- 
1.7.0.4

